portfolio/src/main/webapp/script.js

```diff
@@ -70,8 +71,8 @@ function deleteComments() {
 
   arrayComments.forEach((comment) => {
     const commentElements = Array.from(comment.children);
-    if (commentElements[1].checked == true) {
-        deleteSingleComment(commentElements[1].value);
+    if (commentElements[0].checked == true) {
```

Comment:
This is a bit hard to read, is the comment.children a list of commentElement as in createListElement function?
Could you get the check box by name/id/class instead of by index?

Reply:
I found a way to get the checkbox by name, as a result the code is simpler and easier to read. I added and committed that change.